oca_port/cli/main.py

```diff
@@ -112,20 +128,24 @@ def main(
             The PRs are found from FROM_BRANCH commits that do not exist in TO_BRANCH.
     The user will be asked if he wants to port them.
 
-        To start the migration process, the `--fork` option must be provided in
+        To start the migration process, the `--destination` option must be provided in
     order to push the resulting branch on the user's remote.
     """
+    source = make_gh_info("source", source)
+    target = make_gh_info("target", target)
+    destination = make_gh_info("destination", destination)
```

Comment:
I would move this in `App.__init__`, it's OK IMO to still parse `source` here to retrieve the repo, it'll be parsed a second time in `App` but that's something we can afford.